src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1566,6 +1566,22 @@ public String toVariableNameString(EnchantmentOffer eo) {
 					"Individual attributes of a snapshot cannot be modified or retrieved.")
 				.requiredPlugins("Minecraft 1.20.2+")
 				.since("INSERT VERSION")
+				.parser(new Parser<>() {
+					@Override
+					public boolean canParse(ParseContext context) {
+						return false;
+					}
+
+					@Override
+					public String toString(EntitySnapshot snapshot, int flags) {
+						return snapshot.getEntityType().toString().toLowerCase(Locale.ENGLISH).replace('_', ' ') + " snapshot";
```

Comment:
I wonder if we might get friendlier names from converting to an EntityData and stringifying that?